SQL database/Villagers.sql

```diff
@@ -0,0 +1,87 @@
+CREATE DATABASE village;
+USE village;
+-- //creating tables
+
+CREATE TABLE IF NOT EXISTS villagers (
+    villager_id INT AUTO_INCREMENT PRIMARY KEY,
+    first_name VARCHAR(50) NOT NULL,
+    last_name VARCHAR(50) NOT NULL,
+    user_name VARCHAR(50) NOT NULL,
+    birthday DATE NOT NULL,
+    email VARCHAR(100) NOT NULL UNIQUE,
+	villager_address TEXT NOT NULL,
+    villager_postcode TEXT NOT NULL,
+	villager_location ENUM('North London', 'South London', 'West London', 'East London'),
+    password VARCHAR(255) NOT NULL
+);
+  
+  CREATE TABLE IF NOT EXISTS children (
+	child_id INT AUTO_INCREMENT PRIMARY KEY,
+	parent_id INT NOT NULL,
+	nick_name ENUM('Walnut', 'Pistachio', 'Cashew') NOT NULL,
+	birthday DATE NOT NULL,
+	FOREIGN KEY (parent_id) REFERENCES villagers(villager_id) ON DELETE CASCADE
+
+);
+
+ CREATE TABLE IF NOT EXISTS events (
+    event_id INT AUTO_INCREMENT PRIMARY KEY,
+    event_name VARCHAR(50) NOT NULL,
+    event_description TEXT NOT NULL,
+    event_location ENUM('North London', 'South London', 'West London', 'East London'),
+	event_address TEXT NOT NULL,
```

Comment:
Extra space before `event_address` - that space might not be showing on vscode, the way for you to check it is to highlight the lines on vscode and check if instead of dots in front of the line you see an arrow and spaces (like on line 32 below):

<img width="546" alt="image" src="https://github.com/user-attachments/assets/c549b304-a7ab-48d2-b1b4-34fa97b42644">